### ISSUE TITLE
[Bug] Item table context menus not properly accessible through keyboard/screen reader

### ISSUE THREAD

**Issue by CaveMobster** [crowbartools/Firebot#1279]
**Describe the bug**
Context menus in the firebot item table cannot be opened with screen readers/keyboard. The menu gets dynamically generated and then appended to the body in the DOM, which is not a good solution for screen readers and we would have to make workarounds that can result in unexpected side effects.
